examples/rstudio/compare.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Maybe this should be a function in `common.bash` instead of a stand-alone script?

Reply:
Changed the implementation so we don't need this script anymore.